pawnShop/src/main/java/com/example/pawnShop/Controller/ProductTypeController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.pawnShop.Controller;
+
+import com.example.pawnShop.Dto.Product.ProductTypeDto;
+import com.example.pawnShop.Service.Contract.ProductTypeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/product_type")
+public class ProductTypeController {
+    private final ProductTypeService productTypeService;
+
+    @GetMapping()
+    public ResponseEntity<List<ProductTypeDto>> getAllProductsTypes(){
+        List<ProductTypeDto> productsTypes = productTypeService.getAllProductsTypes();
+
+        return ResponseEntity.ok(productsTypes);
+    }
+    @GetMapping("/{id}")
+    public ResponseEntity<ProductTypeDto> getProductTypeById(@PathVariable UUID id){
+        ProductTypeDto productType = productTypeService.getProductTypeById(id);
+
+        return ResponseEntity.ok(productType);
+    }
+    @PutMapping("/{newProductTypeName}")
+    public ResponseEntity<String> addProductType(@PathVariable String newProductTypeName){
+        String productTypeName = productTypeService.addProductType(newProductTypeName);
+
+        return ResponseEntity.ok(productTypeName);
+    }
+    @PatchMapping("/{id}")
+    public ResponseEntity<String> modifyProductType(@PathVariable UUID id, @RequestBody ProductTypeDto modifiedProductType){
+        String modifiedProductTypeName = productTypeService.updateProductType(id, modifiedProductType);
+
+        return ResponseEntity.ok(modifiedProductTypeName);
+    }
+    @DeleteMapping("/{id}")
+    public ResponseEntity<String> deleteProductType(@PathVariable UUID id){
```

Comment:
Return the whole object always. Returning scalars does not give any information to the FE at all. 
Check in postman:
Make delete request and like that you will get only 1 string. Imagine if FE wants to render other data from the deleted entity. Then we will need to modify it again 😀. In all cases, please, it is a good practice to not return scalars, instead return thr whole object